frontend/app_flowy/packages/appflowy_editor/test/infra/test_raw_key_event.dart

```diff
@@ -139,6 +139,12 @@ extension on LogicalKeyboardKey {
     if (this == LogicalKeyboardKey.keyZ) {
       return PhysicalKeyboardKey.keyZ;
     }
+    if (this == LogicalKeyboardKey.asterisk) {
+      return PhysicalKeyboardKey.digit8;
+    }
+    if (this == LogicalKeyboardKey.underscore) {
+      return PhysicalKeyboardKey.minus;
```

Comment:
I'm not sure what should be here, could you advise with that?

Reply:
The `minus` is fine. Since I often encountered some error reports, like https://github.com/flutter/flutter/issues/96022, when simulating keyboard pressing in the test before, I didn't use the Flutter built-in keyboard testing function directly but passed the keyboard event to the `keyboard_service`. The mock keyboard events need to correspond to the logical keys and physical keys, so there is no problem to use `minus` here.